### DISCUSSION TITLE
upstream request timeout

### DISCUSSION THREAD

**Discussion by petermacaulay** [Budibase/budibase#2948] [General]
During app build I get 'upstream request timeout'. 

I am using DigitalOcean in Singapore.
/PETER MAC

**Comment by mjashanks** [comment]
Hi Peter.

Could you give some more details? 
- Where are you seeing this error? 
- Can you give a screenshot?
- What action did you take to produce the error?

Also, our digital ocean droplet is quite out of date. You might have to manually update it to get to the latest version. It's discussed in here: https://github.com/Budibase/budibase/discussions/1633
